src/listeners/client/guild/guildDelete.ts

```diff
@@ -8,15 +8,11 @@ import { Guild, time } from 'discord.js';
 @ApplyOptions<ListenerOptions>({ event: Events.GuildDelete })
 export class UserEvent extends Listener<typeof Events.GuildDelete> {
 	public async run(guild: Guild) {
-		const guildId = guild.id;
-
-		if (!container.client.isReady()) return;
-
-		this.container.logger.debug(`[EVENT] ${Events.GuildDelete} - ${guild.name} (${guildId})`);
-		DEBUG ? container.logger.debug(`[GuildDelete] - ${guild.toString()}`) : null;
+		if (guild.available) return;
 
+		this.container.client.guildMemberFetchQueue.remove(guild.shardId, guild.id);
 		await this.leaveServerLog(guild);
-		await getSettings(guildId).update({ disabled: true, inviter: null });
```

Comment:
this disabled update was done on purpose so that owners that kicked the bot on accident can reinvite the bot without loosing any data. after 1 day it will get deleted

Reply:
I don't think this is necessary you can't delete a bot like that if that's the case it was necessary 🤔
Afterwards it's clearly not what takes up the most space in the db so I think that even just deactivating it would already be good and not deleting it.
I think we should clean the database less often but for a larger number so I think that a check only every month could already be good.

Reply:
I agree, doing it every day might be a bit much, the deletion of the  data is more of a "privacy" feature, that people can check if we really delete the data once they removed the bot! 

Let's disable the guilds on bot removal and delete all disabled guilds plus their birthdays once a week! Sounds good?

Reply:
@Swiizyy can we do i like this?

Reply:
Yes 👍